go.mod

```diff
@@ -157,15 +157,30 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-require github.com/envoyproxy/protoc-gen-validate v0.10.1
+require (
+	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
```

Comment:
This seems to re-add the "old" Azure SDK, which is deprecated and we want to avoid. Is there a way to fetch it using the "new" SDK?

Reply:
Not at the moment.